src/main/java/helpers/PropertiesHelper.java

```diff
@@ -0,0 +1,41 @@
+package helpers;
+
+import java.io.*;
+import java.util.Properties;
+
+public class PropertiesHelper {
+
+    public String propertiesReader(String prop, String propFileName) {
+        Properties properties = new Properties();
+        InputStream inputStream;
+
+        File file = new File("src/main/resources/authCreds.properties" + propFileName);
+        try {
```

Comment:
try (InputStream inputStream=new FileInputStream(file); ) {
}

Reply:
Could I clarify why is that? I already have a declared variable:

InputStream inputStream;

Reply:
It is just about to clean up a bit code. Such construction called try with resources which allows you to reduce a bit code. More about it :https://www.baeldung.com/java-try-with-resources

Reply:
Got it! Great tip, thanks! Refactored